src/routes/users/index.js

```diff
@@ -17,9 +18,17 @@ const ERROR_MAP = {
 
 const router = express.Router()
 
-const fetchUsers = (req, res) => {
-	res.send("get users")
-}
+const fetchUsers =  catchAsync( async(req, res) => {
+	const users = await userService.fetchUsers()
+
+	res.status(200).json({
+		status: 'success',
+		data: users.map((user) => (
+			transformUserResponse(user)
+		))
```

Comment:
what we already have works great so this a refactor suggestion, not a necessity but just for learning purposes

```suggestion
		data: users.map(transformUserResponse)
```